packages/distlib/meta.yaml

```diff
@@ -1,11 +1,14 @@
 package:
   name: distlib
-  version: 0.3.1
+  version: 0.3.4
 source:
-  sha256: edf6116872c863e1aa9d5bb7cb5e05a022c519a4594dc703843343a9ddd9bff1
-  url: https://files.pythonhosted.org/packages/2f/83/1eba07997b8ba58d92b3e51445d5bf36f9fba9cb8166bcae99b9c3464841/distlib-0.3.1.zip
+  sha256: 6564fe0a8f51e734df6333d08b8b94d4ea8ee6b99b5ed50613f731fd4089f34b
+  url: https://files.pythonhosted.org/packages/ac/a3/8ee4f54d5f12e16eeeda6b7df3dfdbda24e6cc572c86ff959a4ce110391b/distlib-0.3.4-py2.py3-none-any.whl
+  finished_wheel: True
```

Comment:
Do we need this parameter? We could pretty much look at the extension and the file ending with `none-any` couldn't we ?

Reply:
Yeah, that's definitely possible, perhaps that'd be better. After we've merged this PR we should change all the unpatched pure Python packages to use this method.

Reply:
Even with patches, we could potentially download wheel, unpack wheel, apply patch, repack wheel. Then again, if we do this to avoid handling the alternate build backends used by `typing_extensions` and `distlib`, maybe we'd be better off actually figuring out how to handle alternate build backends.

Reply:
Also, would be good to update `mkpkg` to prefer to take pure Python wheels over sdists. Maybe with a flag to indicate that we actually want an sdist. For updates, can default to keeping the same type.